### PR TITLE
Removed hardcoded country summary in presenter.

### PR DESCRIPTION
**Pull request by dsjen**
@rocha @clintonb

### CODE REVIEW

File: analytics_dashboard/courses/presenters.py [hunk 1]
```diff
@@ -142,7 +142,8 @@ def get_geography_data(self):
                 'num_countries': len(data),
                 'top_countries': []
             }
-            for i in range(0, 3):
+            country_display_max = 3
```
Comment (clintonb) [inline analytics_dashboard/courses/presenters.py]: The variable isn't really necessary.

**Comment by clintonb** [comment]
:+1:

File: analytics_dashboard/courses/presenters.py [hunk 2]
```diff
@@ -142,7 +142,8 @@ def get_geography_data(self):
...
+            for i in range(0, min(country_display_max, len(data))):
                 summary['top_countries'].append(data[i])
```
Comment (rocha) [inline analytics_dashboard/courses/presenters.py]: A little bit more pythonic:

```
MAX_COUNTRY_DISPLAY = 3  # as a module variable.

...

summary['top_countries'].extend(data[:MAX_COUNTRY_DISPLAY])


```

Comment (dsjen) [inline analytics_dashboard/courses/presenters.py]: It's been refactored to use `MAX_COUNTRY_DISPLAY` and set `top_countries` immediately in summary.

File: analytics_dashboard/courses/presenters.py [hunk 3]
```diff
@@ -140,10 +142,8 @@ def get_geography_data(self):
             summary = {
                 'last_updated': last_updated,
                 'num_countries': len(data),
-                'top_countries': []
+                'top_countries': data[:self.MAX_COUNTRY_DISPLAY]
```
Comment (rocha) [inline analytics_dashboard/courses/presenters.py]: Super minor: something like `NUMBER_TOP_COUNTRIES` may be a better name.

**Comment by rocha** [comment]
:+1:

File: analytics_dashboard/courses/presenters.py (hunk 3 above)
Comment (dsjen) [inline analytics_dashboard/courses/presenters.py]: Updated the name.

**Pull request merged**